src/NuGet.Services.EndToEnd/Support/Clients/RegistrationClient.cs

```diff
@@ -164,11 +164,21 @@ public RegistrationClient(SimpleHttpClient httpClient, V3IndexClient v3IndexClie
         private class RegistrationIndexResponse
         {
             public List<RegistrationPage> Items { get; set; }
+
+            public RegistrationIndexResponse()
+            {
+                Items = new List<RegistrationPage>();
```

Comment:
I would initialize property inline instead of adding a constructor, it is allowed since C# 6, IIRC.
```
public List<RegistrationPage> Items { get; set; } = new List<RegistrationPage>();
```